packages/taikoon-ui/src/components/core/Text/H0.svelte

```diff
@@ -0,0 +1,12 @@
+<script lang="ts">
+  import { classNames } from '$lib/util/classNames';
+
+  import { CommonTextClasses } from './common';
+
+  let classes: string = null;
+  export { classes as class };
+</script>
+
+<div class={classNames(CommonTextClasses, 'font-medium', 'text-h0', classes)}>
```

Comment:
Two things:

- Why not use plain CSS? Just use tailwind @apply rules for your class. This just creates bloat.
- You can access classes via svelte `$$props.class`, no need to export it. See: https://svelte.dev/repl/6de3c939fcbf4c90a0309210c04e1c3a?version=3.17.2